warehouse/accounts/views.py

```diff
@@ -215,6 +216,13 @@ def register(request, _form_class=RegistrationForm):
     if request.authenticated_userid is not None:
         return HTTPSeeOther("/")
 
+    if AdminFlag().is_enabled(request.db, 'disallow-new-user-registration'):
+        request.session.flash(
+            "User Registration Temporarily Disabled",
+            queue="error",
+        )
+        raise HTTPForbidden()
```

Comment:
Could be wrong here but I'm not sure the flashed message will appear on the 403 page.

Reply:
hrm... yeah. good catch.

Reply:
We _certainly_ need to message correctly to users what's up whenever we use these. I'm going to see what options we have.

Reply:
Could just redirect to a page that shows the flashed messages (like the index).

Reply:
Addressed!